CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android Drop-In Release Notes
 
+## unreleased
+
+* Bump braintree_android version to 3.14.2
```

Comment:
should we add a note that this update fixes the issue with the cardinal SDK?

Reply:
Most def yeah. We closed the issue as a duplicate and re-directed everyone to `braintree-android` for status updates, but we can def reference it here.

Reply:
I think in general we try to keep the CHANGELOG entries specific to changes in this SDK. Merchants can reference the CHANGELOG of the other repo to see what changed there.

Reply:
I see the concern though since the Cardinal issue was pretty severe

Reply:
We've added a link back to the original drop in issue for consistency. That issue then links to the currently open issue in braintree-android, so anyone interested can find the full conversation.